### ISSUE TITLE
gencli: Unused strings import

### ISSUE THREAD

**Issue by ericwenn** [googleapis/gapic-generator-go#333]
When generating CLI for a message with an `OUTPUT_ONLY` enum field `strings` is imported but not used, leading to a broken build.

Example message:
```proto
message Message {
  enum State {
    STATE_UNSPECIFIED = 0;
    STATE_ACTIVE = 1;
  }
  State state = 1 [(google.api.field_behavior) = OUTPUT_ONLY];
}
```

Either also check if enum is output only before inserting import, or fake-reference the strings pkg à la  `var _ = strings.Contains`.

**Comment by noahdietz** [comment]
Fix released in v0.12.2. Check it out. Thanks for continuing to report bugs and using the tool! Let me know if there is anything else.

**Comment by ericwenn** [comment]
Awesome! Thanks for fixing this so quickly. Will let you know if I find some more :)
